godot-macros/src/class/derive_godot_class.rs

```diff
@@ -592,3 +608,11 @@ fn is_class_virtual_extension(godot_class_name: &str) -> bool {
         _ => godot_class_name.ends_with("Extension"),
     }
 }
+
+/// Whether a class exists primarily as a plugin for the editor.
+// See post_validate(). Should be moved to godot-codegen > special_cases.rs.
+// This information is available in extension_api.json under classes.*.api_type and should be taken from there.
+fn is_class_editor_only(godot_class_name: &str) -> bool {
+    // Checking the prefix covers the primary chunk of classes marked as editor only.
+    godot_class_name.starts_with("Editor")
```

Comment:
I'm surprised, this covers [a lot of classes](https://godot-rust.github.io/docs/gdext/master/godot/?search=Editor)! 🙂 

In the future, a more robust way would be to check for the JSON key-value
```json
"api_type": "editor"
```

Could you add a TODO here?

Also, there are a few more that you can add manually now:
- `FileSystemDock`
- `ScriptEditor`
- `ScriptEditorBase`
- `ScriptCreateDialog`

Reply:
The remaining classes should now also be covered.